pkg/controller/ovn_eip.go

```diff
@@ -364,17 +364,19 @@ func (c *Controller) createOrUpdateOvnEipCR(key, subnet, v4ip, v6ip, mac, usageT
 	return nil
 }
 
-func (c *Controller) patchOvnEipStatus(key string, ready bool) error {
+func (c *Controller) patchOvnEipStatus(key string, ready, changeReady bool) error {
```

Comment:
how about removing  `ready, changeReady` and eip status `ready`,  keep the ovn-eip just as IP crd？ keep it simple